src/components/Training/Approach.js

```diff
@@ -5,6 +5,8 @@ import { SectionTitle, Subtitle, BodyPrimary } from '../Typography'
 import { Grid, Row, Col } from '../grid'
 import Hr from '../Common/Hr'
 import BlueBG from '../BlueBG'
+import TitleAndList from '../Common/TitleAndList'
+
 
 const Approach = ({ title, content, formats }) => (
   <BlueBG>
```

Comment:
I think we should include the background color in the TitleAndList component itself. Otherwise you can surround it with a BlueBG component & not pass in the `dark` prop. I think these two things (colour of the text & background color) should be inextricably linked within the component itself.

This is where I would suggest that we
1. Delete the `GreyBG` & `BlueBG` components altgoether
2. Alter `Grid` to take in a `bg` prop & use this instead

Just mentioning this now, while we're introducing the potential scenario described above, as a discussion point (not necessarily a requested change on this PR).

Reply:
'Paiting' each component doesn't seem like a good solution for this. This type of components are usually transparent (`background: none`) and simply reflect their context's background. Why would we need to add another layer?

> Otherwise you can surround it with a BlueBG component & not pass in the dark prop.

Wouldn't this happen with your suggestion? Surrounding the component with another background and not passing the `dark`/bg prop?

IMO, the context/`mode` prop should be enough to reflect the designer's desired contrast (ie. change the text color), don't you think?

Btw, this isn't related to the `greyBg`/Grid suggestion you're referring